charts/publisher/templates/web/slack-notify-sync.yaml

```
@@ -0,0 +1,57 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-slack-msg-success
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      containers:
+      - name: slack-notification
+        image: curlimages/curl
+        command:
+          - "curl"
+          - "-X"
+          - "POST"
+          - "--data-urlencode"
+          - "payload={\"channel\": \"#govuk-deploy-alerts\", \"username\": \"argocd\", \"text\": \"{{ .Release.Name }} sync succeeded (image_tag: {{ .Values.common.image.tag }}) :deployparrot:\", \"icon_emoji\": \":kubernetes:\"}"
```

Consider using the block scalar syntax here for readability, e.g. something like
```suggestion
          - >
              payload={
                "channel": "#govuk-deploy-alerts",
                "username": "argocd",
                "text": "{{ .Release.Name }} sync succeeded (image_tag: {{ .Values.common.image.tag }}) :deployparrot:",
                "icon_emoji": ":kubernetes:"
              }
```

---

Thank _you_ I wasn't aware of this YAML syntax.